### DISCUSSION TITLE
Cannot install NVidia drivers in OS 7 Horus

### DISCUSSION THREAD

**Discussion by robinemery76** [elementary/appcenter#1984] [Q&A]
I am able to successfully install EOS 7 on my production machine, however I am unable to successfully install and run with any of the NVidia drivers suggested in the Updates section of the AppCenter.

I can click to install. It appears to install something, as when I run "whereis nvidia" in a terminal, there is evidence that something is installed yet the AppCenter still suggests I need/can update the driver of 470, 515 and 525 (latest at time of install).

Is anyone else experiencing the same issue on their NVidia install?

I have an MSI NVidia GeForce GTX 1050Ti with 4GB RAM, installed on Intel i5 7th Gen @ 3GHz with 16GB RAM.

Thank you!

**Comment by robinemery76** [comment]
In reply to this, I have been successful to install EOS 7 on my laptop and Intel i3 mini PC running Mesa graphics. Runs smoothly with no issues on this device.

**Reply by zietbukuel** [reply]
What did you do to make it work? I've not been able to install it using the appstore, I had to install it manually with apt.
